hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/server/ServiceRuntimeInfo.java

```diff
@@ -33,6 +33,15 @@
  */
 public interface ServiceRuntimeInfo {
 
+  /**
+   * Gets the namespace of Hadoop.
```

Comment:
nit: reference to Hadoop in existing comments is outdated; no need to fix that, but let's not repeat it. :)